lib/travis/build/addons/chrome.rb

```diff
@@ -0,0 +1,72 @@
+require 'shellwords'
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Chrome < Base
+        SUPER_USER_SAFE = true
+
+        attr_reader :version
+
+        def after_prepare
+          sh.fold 'install_chrome' do
+            sanitize(raw_version)
```

Comment:
This is def a nitpick, but instead of `sanitize(raw_version)` with later accessing `version`, why not implement version as `@version ||= <sanitizing-code-with-access-to-raw_version>`?